OCaml.tmLanguage

```diff
@@ -1626,7 +1626,7 @@
 			<array>
 				<dict>
 					<key>begin</key>
-					<string>(val!?)(\s+mutable)?\s+([a-z_][a-zA-Z0-9_']*|\([=&lt;&gt;@^&amp;+\-*/$%|][|!$%&amp;*+./:&lt;=&gt;?@^~-]*\))\s*(:)</string>
+					<string>(val!?)(\s+mutable)?\s+([a-z_][a-zA-Z0-9_']*|\([=&lt;&gt;@^&amp;+\-*/$%|][|!$%&amp;*+./:&lt;=&gt;?@^~-]*\))\s*(:?)</string>
```

Comment:
That's the wrong fix (the regex should capture just as much after the fix, not less; not sure why \n is trouble), and I can't even pick up the other one because they're in the same commit.